app.js

```diff
@@ -32,6 +35,9 @@ app.use('/api/upgrade7/backup', upgrade7Backup);
 app.use('/api/upgrade7/admin-repo-checks', upgrade7AdminRepoChecks);
 app.use('/api/upgrade7/admin-upgrade', upgrade7AdminUpgrade);
 app.use('/api/upgrade7/nodes-repo-checks', upgrade7NodesRepoChecks);
+app.use('/api/upgrade7/openstack-services', upgrade7OpenStackServices);
+app.use('/api/upgrade7/stop-openstack-services', upgrade7StopOpenStackServices);
```

Comment:
This could be updated to `/api/upgrade7/openstack-services/stop` and `/api/upgrade7/openstack-services/backup`